src/cases/containers/allocate-role-container/allocate-role-container.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
+
+import * as fromRoot from '../../../app/store';
+import {RoleAllocationType } from '../../../cases/enums/allocation-type';
+
+@Component({
+  selector: 'exui-allocate-role-container',
+  templateUrl: './allocate-role-container.component.html'
+})
+export class AllocateRoleContainerComponent implements OnInit {
+
+  public roleAllocationType: RoleAllocationType;
```

Comment:
would this best to move to src\work-allocation-2\containers, if this is only used in work allocation 2.